load/UpdateDBPVideoTables.py

```diff
@@ -184,15 +184,15 @@ def computeDurations(self, hashId):
 
 
 if (__name__ == '__main__'):
-	from LPTSExtractReader import *
+	from LanguageReader import *
 	from InputFileset import *
 	from DBPLoadController import *
```

Comment:
we should import the LanguageReaderCreator in the main to execute the UpdateDBPVideoTables.py file locally. e.g.

```python
from LanguageReaderCreator import LanguageReaderCreator
```